west.yml

```diff
@@ -258,7 +258,7 @@ manifest:
       groups:
         - hal
     - name: hal_wch
-      revision: 1de9d3e406726702ce7cfc504509a02ecc463554
+      revision: pull/1/head
```

Comment:
Is this correct?

Reply:
I think, it's to indicate that this PR depend on the PR \#1 on the hal_wch repository. The message from zephyrbot at the top of the PR looks good so I think it's correct.